.NET/Microsoft.Recognizers.Text.DateTime/Spanish/Parsers/SpanishDatePeriodParserConfiguration.cs

```diff
@@ -191,7 +191,7 @@ public SpanishDatePeriodParserConfiguration(ICommonDateTimeParserConfiguration c
 
         Regex IDatePeriodParserConfiguration.ThisPrefixRegex => ThisPrefixRegex;
 
-        Regex IDatePeriodParserConfiguration.RelativeRegex => RelativeRegex;
+        Regex ISimpleDatePeriodParserConfiguration.RelativeRegex => RelativeRegex;

```

Comment:
You should not change the interfaces everywhere. Especially mixing types. Please keep the original ones.

Reply:
My understanding from your code is that ISimpleDatePeriodParserConfiguration defines just the common interface needed by your new method. If that's the case, you can make the other interfaces that should fir there to inherit from this simple interface.

Reply:
I see now this is just specifying the property declaration. Please ignore this comment.